### DISCUSSION TITLE
Server location

### DISCUSSION THREAD

**Discussion by LucaKrummi** [community/community#32188] [Code Security] [answered]
Hi there

We are currently talking about using GitHub for our company to develop software. (I am familiar with it, but my company isn't).
My boss is kinda sceptical about the safety of our data.
Our firm is in Europe.
Does anyone know where the location of the servers are?
Is the data stored in Europe or all over the world?

Thank you for your answer and kind regards

Luca

**Comment by xinxinxiangying258** [comment]
you could try to build an on-schedule workflow for testing.
I found that github may use UTC, but somehow unstable(several minutes late?) which may because that the action was queueing in the list waiting for a machine to run this workflow...

**Comment by jhutchings1** [comment, marked as answer]
Our sales team can answer these questions for you. https://github.com/enterprise/contact
